source/exoscale-vouchers.rst

```diff
@@ -0,0 +1,161 @@
+Exoscale Vouchers for AoD 
+=========================
+
+In this section we describe how users can easily discover the `EGI
+Applications on Demand
+(AoD) <https://www.egi.eu/services/applications-on-demand/>`_ service
+published in the `EOSC portal
+Marketplace <https://marketplace.eosc-portal.eu/>`_ and submit a
+service-order request to deploy an elastic virtual cluster on the
+HNSciCloud pilot services. The virtual cluster will be deployed and
+configured with the requested software packages using the `Elastic
+Cloud Compute Cluster
+(EC3) <https://servproject.i3m.upv.es/ec3-ltos/>`_ portal.
+
+
+Submit a service-order request through the EOSC Portal Marketplace
+------------------------------------------------------------------
+
+The `EOSC Portal Marketplace <https://marketplace.eosc-portal.eu/>`_ is a
+single entry point where users can navigate the EOSC service catalogue,
+discover a service of interest, get information about it, and place an order
+in a few clicks by specifying the service requested along with the quantity,
+quality and duration.   
+
+All the available services of the EOSC Marketplace are grouped in 8 main 
+categories and no authentication is required to browse and discover the
+list of services published in the marketplace.
+
+Exoscale vouchers can be used when selecting he Elastic Cloud Compute Cluster
+(EC3) platform from the list of available services by submitting a service-order
+request.
+
+.. image:: img/ec3-eosc-marketplace.png
+   :width: 900px
+   :alt: The Elastic Cloud Compute Cluster (EC3) in the EOSC Portal Marketplace.
+
+Placing orders in the EOSC Marketplace requires the user to authenticate. There
+are two login options: use your personal academic credentials or use EGI Check-in
+service. The first time you login into the marketplace requires some extra
+information to be provided to configure the marketplace account.
+
+Two different service options are available for EC3:
+
+.. image:: img/ec3-service-options.png
+   :width: 900px
+   :alt: The Elastic Cloud Compute Cluster (EC3) service options.
+
+To deploy the virtual cluster on the HNSciCloud pilot services, select the
+voucher option and click on *Next* and provide additional information for
+profiling the order (e.g. the motivation for requesting the service).
+
+**This makes little sense**
+If the user has already an Exoscale voucher to redeem, the voucherID
+can be provided by the user during the submission of the
+service-order request, otherwise a valid voucherID will be provided
+by the operator of the service in case the user’s vetting process is
+successful.
```

Comment:
@glarocca I didn't understand what is supposed to be said here

Reply:
I meant that we can offer two options. 
- When user submits a service-order request through the MP he/she can request access to the EC3 framework and provide a voucherID if he/she already have one (he could get one during one of the past HNSciCloud promotional events). 
- User can submit a service-order request to use the EC3 framework with voucher. In this case, the operator on shift can provide a voucherID to the user in case the vetting process is successful. VoucherID will be obtained contacting the HNSciCloud PO.

Reply:
I rephrased, let me know if everything is ok

Reply:
Ok, fine with me

Reply:
@glarocca can you then approve the PR?